docker-cloud/apps/stack-yaml-reference.md

```diff
@@ -47,7 +47,7 @@ The image used to deploy this service. This is the only mandatory key.
 ```yml
 image: drupal
 image: dockercloud/hello-world
-image: quay.io/borja/redis
+image: my.registry.com/redis
```

Comment:
It's an example domain. If we are worried about it not working with `my.registry.com/redis`, I'd remove it altogether.